app/models/drupal_node_revision.rb

```diff
@@ -43,6 +43,13 @@ def inline_tags
     end
   end
 
+  # search for inline hashtags(such as #hashtag) and create a new tag
+  def inline_hashtags
+    self.body.scan(/[^\n]\#([\w-]+)/).each do |match|
```

Comment:
Hey @chongr I think this regexp won't give correct results. It checks hashtags only at the beginning of new line. The correct hashtag would be as used in the constants file. Refer to this https://github.com/publiclab/plots2/blob/master/config/initializers/constants.rb#L3. The best way would be to use `Callouts.const_get(:HASHTAG)` for the hashtag regexp as used elsewhere in the application.